opengeode/AdaGenerator.py

```diff
@@ -528,7 +528,7 @@ def _process(process, simu=False, instance=False, taste=False, **kwargs) -> str:
         ads_template.append(f'procedure Startup;')
 
 
-    # Generate the the code of the procedures
+    # Generate the code of the procedures
```

Comment:
I'm still playing w/ duplicate word detection. I think this class of changes is correct.